### DISCUSSION TITLE
Show chords without capital letter M for Major

### DISCUSSION THREAD

**Discussion by martinjnilsen** [tonaljs/tonal#277] [Q&A]
Hi.

Is there a way to not include the capital letter "M" for major chords?

**Comment by danigb** [comment]
Can you elaborate a little, please?

**Comment by martinjnilsen** [comment]
When I worked on the project our customer wanted the chords C-major, D-major etc. to not include the capital letter "M". As I used the tonaljs library, the M was used for indicating that it was the major chord, in which I wanted to know if it was an built in solution for not including. I solved it by removing the capital M while looping through all the cords, but feel like it maybe could have been done better. Its been a month since I finished up the prototype, but please provide me with a better solution if it exists already!

**Comment by danigb** [comment]
Sorry, I don't have any better solution. Thanks for the explanation.
